moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -14,35 +15,25 @@ import {
   ProjectComponentsSourcesAndLayers,
 } from "./mapUtils";
 import "mapbox-gl/dist/mapbox-gl.css";
+import { isDrawnFeature, makeCapturedFromLayerFeature } from "./featureUtils";
 
 // See https://github.com/visgl/react-map-gl/issues/1266#issuecomment-753686953
 import mapboxgl from "mapbox-gl";
 mapboxgl.workerClass =
   // eslint-disable-next-line import/no-webpack-loader-syntax
   require("worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker").default;
 
-const deDeupeProjectFeatures = (features) => {
```

Comment:
I removed this since the IDs checked here don't agree with the unique IDs of drawn features, and they were hidden as a result. I can spin off an issue to refactor this or include in this PR. 

Just realized that I forgot to circle back on this.

Reply:
👍 this removes the ability to see overlapping features in the popups, but that functionality has not been well scoped and i'm happy to flesh out that behavior in [this issue](https://github.com/cityofaustin/atd-data-tech/issues/10715).